rules-reviewed/eap6/weblogic/weblogic-webapp.windup.xml

```diff
@@ -100,7 +100,7 @@
                       See the [javax.servlet.annotation JavaDoc](http://docs.oracle.com/javaee/6/api/javax/servlet/annotation/package-summary.html) for more information.
                       ]]>
                     </message>
-                    <link href="https://access.redhat.com/articles/1249423" title="Migrate WebLogic Proprietary Servlet Annotations" />
+                    <link href="https://access.redhat.com/articles/1249423" title="WebLogic proprietary servlet annotations" />
```

Comment:
Ok! It is fine to keep them in general. Sometimes the original titles are quite terrible or have Too Many Words Starting With A Capital Letter ;-)